app/src/renderer/components/wallet/PageTransactions.vue

```diff
@@ -43,7 +43,8 @@ export default {
   computed: {
     ...mapGetters(['filters', 'transactions', 'wallet', 'config']),
     orderedTransactions () {
-      return orderBy(this.transactions, [this.sort.property], [this.sort.order])
+      let list = orderBy(this.transactions, [this.sort.property], [this.sort.order])
+      return uniqBy(list, 'time') // filter out duplicate tx to self
```

Comment:
Correct me, but I think this would squash all tx to yourself by block (which might be ok, as it's a rare case and blocks are proposed every 1s?).

Reply:
Ah you're right. That's not intended. Will fix.

Reply:
I looked into resolving this and I think we need to wait until transactions have uids before this can be correctly fixed. Any sort of uniqueness filtering is stymied by the fact that the use can send two self transactions with exactly the same amount of tokens in the same block.